backend/app/views/backoffice/open_calls/edit.html.erb

```diff
@@ -20,6 +20,10 @@
     <% end %>
   </aside>
   <div class="w-full h-full bg-white rounded-xl p-6">
+    <div class="mb-3 flex justify-end gap-3 text-s text-gray-600">
+      <span>Created: <%= I18n.l @open_call.created_at.to_date %></span>
```

Comment:
Not sure if `Created:` and `Updated:` text is so international that it does not need to be translated, but maybe it should be moved to `I18n`.

Reply:
of course 🙈  - thanks!